1001-2000/LIT1501GadlaS.xml

```diff
@@ -84,7 +84,6 @@ each devoted to a particular miracle.</p>
                <listRelation>
                    <relation name="ecrm:P129_is_about" active="LIT1501GadlaS" passive="PRS8823Shenute"/>
                    <relation name="saws:isAttributedToAuthor" active="LIT1501GadlaS" passive="PRS2743Besa"/>
-                   <relation name="saws:isVersionOf" active="LIT1501GadlaS" passive="LIT2492VitaSi"/>
```

Comment:
since we delete 2492 we need the relation `betmas:formerlyAlsoListedAs` here, thank you!